actix-session/src/storage/redis_rs.rs

```diff
@@ -259,8 +322,14 @@ impl RedisSessionStore {
     async fn execute_command<T: FromRedisValue>(&self, cmd: &mut Cmd) -> RedisResult<T> {
         let mut can_retry = true;
 
+        #[cfg(feature = "redis-rs-session")]
+        let client = &mut self.client.clone();
+
+        #[cfg(not(feature = "redis-rs-session"))]
+        let client = &mut self.pool.get().await.unwrap();
+
         loop {
-            match cmd.query_async(&mut self.client.clone()).await {
+            match cmd.query_async(client).await {
```

Comment:
here we have a `cmd` shadowing the `use ...redis::cmd`

Reply:
That's true, I've already reversed the changes to that part and I've already applied your suggestion of making _aliases_ to the imports.